docs/manual/docs/cmd/spo/user/user-list.md

```diff
@@ -0,0 +1,36 @@
+# spo user list
+
+Lists all the users within specific web
+
+## Usage
+
+```sh
+spo user list [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --webUrl <webUrl>`|URL of the web to list the users from
+`-o, --output [output]`|Output type. `json,text`. Default `text`
+`--query [query]`|JMESPath query string. See [http://jmespath.org/](http://jmespath.org/) for more information and examples
+`-o, --output [output]`|Output type. `json,text`. Default `text`
```

Comment:
Remove duplicate entries -  Option `--output`

Reply:
To keep the help's contents in sync with the command's help, the first `output` line should be removed rather than this one.